kinto/plugins/kinto_admin/kinto.ini

```diff
@@ -0,0 +1,247 @@
+[server:main]
```

Comment:
We could put that in ``tests/plugins/kinto_admin/config.ini`` since it is a test fixture.